scripts/run-targeted-tests.js

```diff
@@ -0,0 +1,86 @@
+const WRAPPERS = [
+  'angular-handsontable',
+  'react-handsontable',
+  'vue-handsontable',
+];
+const FULL_TEST_BRANCHES = [
+  'master',
+  'develop',
+  'release/',
+];
+
+const { spawnSync, execSync } = require('child_process');
+
+const currentBranch = execSync('git rev-parse --abbrev-ref HEAD', { encoding: 'utf8' }).replace(/(\n)/gm, '');
+const filesModifiedInLastCommit = execSync('git log --name-only --oneline HEAD^..HEAD', { encoding: 'utf8' })
+  .split('\n');
+let hasErrors = false;
+
+const spawnCommand = (command) => {
+  if (hasErrors) {
+    return;
+  }
+  const spawnedCommand = spawnSync('npm', command.split(' '), {
+    stdio: 'inherit'
+  });
+
+  if (spawnedCommand.status) {
+    hasErrors = true;
+  }
+
+  return !hasErrors;
+};
+
+filesModifiedInLastCommit.shift();
+filesModifiedInLastCommit.pop();
+
+const fullTestBranchMatch = currentBranch.match(
+  `(${FULL_TEST_BRANCHES.join('|').replace('/', '\\\\/')}).*`
+);
+
+if (fullTestBranchMatch !== null && fullTestBranchMatch.index === 0) {
+  const status = spawnCommand('run all test');
+
+  if (!status) {
+    process.exitCode = 1;
+  }
+
+  return;
+}
+
+const touchedProjects = [];
+
+filesModifiedInLastCommit.forEach((fileUrl) => {
+  if (fileUrl.includes('CHANGELOG')) {
+    return;
+  }
+
+  const wrappersMatch = fileUrl.match('(wrappers\/)([^\/]*)(\/)');
+
+  if (wrappersMatch) {
+    const projectName = wrappersMatch[2];
```

Comment:
What do you think about explicitly indicating what we want to extract from the string? As the project requires NPM in version 7 and that version is shipped with Node 15 so I let use regex named grouping. 
```suggestion
  const wrappersMatch = fileUrl.match('(wrappers\/)(?<projectName>[^\/]*)(\/)');

  if (wrappersMatch) {
    const { projectName } = wrappersMatch.groups;
```